scraper.rb

```diff
@@ -0,0 +1,20 @@
+require 'pry'
+require 'require_all'
+require 'scraped'
+require 'scraped_page_archive/open-uri'
+
+require_rel 'lib'
+
+# url = 'http://www.althingi.is/altext/cv/en/'
+urls = ['http://www.althingi.is/thingmenn/althingismenn/', # members
+        'http://www.althingi.is/thingmenn/thingmenn/varamenn-sem-sitja-a-althingi/', # alternates
+        'http://www.althingi.is/thingmenn/thingmenn/varamenn-sem-hafa-tekid-saeti/'] # substitues
+
+urls.each do |url|
+  IcelandicMembersPage.new(response: Scraped::Request.new(url: url).response)
+             .member_urls
+             .each do |url|
+               puts "Opening: #{url}"
+               Scraped::HTML.new(response: Scraped::Request.new(url: url).response)
```

Comment:
might be better to assign this to a nil variable (i.e. `_ = Scraped::HTML.new…`) so that it's more obvious that we're simply ignoring the results for now, rather it being a command method.